def raise_exception():
    raise ValueError("Welcome to my Repository, How do I help you")
